plugin.video.dailymotion_com/resources/lib/dailymotion.py

```diff
@@ -0,0 +1,798 @@
+    addDir(translation(30006), "", 'listChannels', _ipath + "channels.png")
+    addDir(translation(30007), urlMain + "/users?fields=username,avatar_large_url,videos_total,views_total&sort=popular&limit=" + itemsPerPage + "&family_filter=" + familyFilter + "&localization=" + language + "&page=1", 'listUsers', _ipath + "users.png")
+    addDir(translation(30002), urlMain + "/videos?fields=description,duration,id,owner.username,taken_time,thumbnail_large_url,title,views_total&search=&sort=relevance&limit=" + itemsPerPage + "&family_filter=" + familyFilter + "&localization=" + language + "&page=1", 'search', _ipath + "search.png")
+    addDir(translation(30002) + " " + translation(30003), "", 'livesearch', _ipath + "search_live.png")
+    addDir(translation(30002) + " " + translation(30007), "", 'usersearch', _ipath + "search_users.png")
+    addDir(translation(30115), "", "History", _ipath + "search.png")
+    if dmUser:
+        addDir(translation(30034), "", "personalMain", _ipath + "my_stuff.png")
+    else:
+        addFavDir(translation(30024), "", "favouriteUsers", _ipath + "favourite_users.png")
+    xbmcplugin.setContent(pluginhandle, "addons")
+    if force_mode:
+        xbmc.executebuiltin('Container.SetViewMode({})'.format(menu_mode))
+    xbmcplugin.endOfDirectory(pluginhandle)
+
+
+def personalMain():
+    addDir(translation(30041), urlMain + "/user/" + dmUser + "/videos?fields=description,duration,id,owner.username,taken_time,thumbnail_large_url,title,views_total&sort=recent&limit=" + itemsPerPage + "&family_filter=" + familyFilter + "&localization=" + language + "&page=1", 'listVideos', _ipath + "videos.png")
+    addDir(translation(30035), urlMain + "/user/" + dmUser + "/following?fields=username,avatar_large_url,videos_total,views_total&sort=popular&limit=" + itemsPerPage + "&family_filter=" + familyFilter + "&localization=" + language + "&page=1", 'listUsers', _ipath + "contacts.png")
+    addDir(translation(30036), urlMain + "/user/" + dmUser + "/subscriptions?fields=description,duration,id,owner.username,taken_time,thumbnail_large_url,title,views_total&sort=recent&limit=" + itemsPerPage + "&family_filter=" + familyFilter + "&localization=" + language + "&page=1", 'listVideos', _ipath + "following.png")
+    addDir(translation(30037), urlMain + "/user/" + dmUser + "/favorites?fields=description,duration,id,owner.username,taken_time,thumbnail_large_url,title,views_total&sort=recent&limit=" + itemsPerPage + "&family_filter=" + familyFilter + "&localization=" + language + "&page=1", 'listVideos', _ipath + "favourites.png")
+    addDir(translation(30038), urlMain + "/user/" + dmUser + "/playlists?fields=id,name,videos_total&sort=recent&limit=" + itemsPerPage + "&family_filter=" + familyFilter + "&localization=" + language + "&page=1", 'listUserPlaylists', _ipath + "playlists.png")
+    xbmcplugin.setContent(pluginhandle, 'addons')
+    if force_mode:
```

Comment:
not allowed sorry. Don't know how this was missed before :/

Reply:
@enen92 
Yes this was raised before and clarified by @Rechi that by default modes can not be set and only set if enabled in settings by user. The code follows that principle and the default is not setting any modes as per that principle
Discussion from #2400 
``` Rechi on May 27 Member
It's okay to allow add a configure setting for forcing the view mode, but it has to be off by default.
```

Reply:
We clarified and it's okay to keep it default off

Reply:
Thanks